src/en/en.json

```diff
@@ -62,6 +62,19 @@
     "releaseNotes": "https://github.com/cds-snc/gcds-components/blob/main/CHANGELOG.md",
     "privacy": "/en/privacy-notice",
 
+    "pageTemplates": "/en/page-templates",
+    "pageTemplatesBasic": "/en/page-templates/basic",
+    "pageTemplatesBasicPreview": "/en/page-templates/basic/preview",
+    "pageTemplatesBasicCode": "/en/page-templates/basic/code",
+    "pageTemplatesBasicGithubLink": "https://github.com/cds-snc/gcds-examples/blob/main/templates/english/basic/basic-page-template.html",
+    "pageTemplatesBasicGithubRaw": "https://raw.githubusercontent.com/cds-snc/gcds-examples/refs/heads/main/templates/english/basic/basic-page-template.html",
+    "pageTemplatesBasicFigma": "#",
+    "pageTemplatesBasicExtOTPPreview": "/en/page-templates/basic/preview-on-this-page",
+    "pageTemplatesBasicExtOTPCode": "/en/page-templates/basic/code-on-this-page",
+    "pageTemplatesBasicExtOTPGithubLink": "https://github.com/cds-snc/gcds-examples/blob/main/templates/english/basic/extensions/basic-page-template-table-of-contents.html",
+    "pageTemplatesBasicExtOTPGithubRaw": "https://raw.githubusercontent.com/cds-snc/gcds-examples/refs/heads/main/templates/english/basic/extensions/basic-page-template-table-of-contents.html",
+    "pageTemplatesBasicExtOTPFigma": "#",
```

Comment:
The figma links in the data files will need to be updated as well

Reply:
I've updated them now, removing the last one too since we don't need it